src/shop-app.html

```diff
@@ -242,21 +234,18 @@
 
     <app-header role="navigation" id="header" effects="waterfall" condenses reveals>
       <app-toolbar>
-        <div class="left-bar-item">
-          <paper-icon-button
-              class="menu-btn"
-              icon="menu"
-              on-tap="_toggleDrawer"
-              aria-label="Categories">
-          </paper-icon-button>
-          <a class="back-btn" href="/list/[[categoryName]]" tabindex="-1">
-            <paper-icon-button icon="arrow-back" aria-label="Go back"></paper-icon-button>
+        <div class="icon-button-container">
+          <a href="#" class="shop-icon-button menu-btn" on-click="_toggleDrawer" aria-label="Categories">
```

Comment:
I don't set any custom focus style - it will show the ring if you tab to it, but otherwise it shouldn't appear.

Here is an example of different anchor attributes. Note that, without `preventDefault`,  `href` will cause a page refresh, `href="#"` changes the URL, and `tabindex="0"` isn't really treated as a link. I think href="#" is closest to what we want, but don't have a strong opinion.